recipes/mcpl/meta.yaml

```
@@ -0,0 +1,68 @@
+package:
+  name: mcpl
+  version: 1.5.0
+
+source:
+  url: https://github.com/mctools/mcpl/archive/v1.5.0.tar.gz
+  sha256: 471322093be60d2f0704fcd9a68756f0f18cf8e5f63a92bd22d8af12fe201ab6
+  #NB get sha256 with: curl -sL <URL> | openssl sha256
+  fn: mcpl-src.tar.gz
+  folder: src
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - zlib
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - zlib
+  run:
+    - python
+    - numpy
+    - zlib
+
+test:
+  requires:
+    - python
+    - numpy
+    - cmake
+    - zlib
```

You don't need to specify run dependencies for test b/c they must be there.
BTW, is cmake really needed here?

```suggestion
    - cmake
```

---

Yes, cmake is needed since one of the tests is verifying that `find_package(MCPL)` works out of the box after the mcpl conda package has been installed.